src/app/components/Unlock/index.jsx

```diff
@@ -54,7 +58,19 @@ class Unlock extends React.Component {
         >
           Unlock
         </Button>
-        <p>{this.state.error}</p>
+        {this.state.error && (
```

Comment:
If there is an decryption error show a message to reset everything. Otherwise the user is stuck and can not do anything.